src/ConcordiumNetSdk/Transactions/AccountTransactionPayload.cs

```diff
@@ -0,0 +1,40 @@
+using ConcordiumNetSdk.Types;
+
+namespace ConcordiumNetSdk.Transactions;
+
+/// <summary>
+/// Account transaction payload.
```

Comment:
This is not a super useful comment.

Don't we also call this "PreAccountTransaction" in the grpc API?